src/qt/unlimiteddialog.cpp

```diff
@@ -192,9 +192,7 @@ void UnlimitedDialog::validateBlockSize()
     int mmb = ui.miningMaxBlock->text().toInt();
     int ebs = ui.excessiveBlockSize->text().toInt();
 
-    /* the mined block size must not be greater then the excessive block 
-     * size else the miner will be mining blocks she cannot track. */
-    if ( mmb > ebs )
+    if ( MiningAndExcessiveBlockValidatorRule(ebs, mmb))
```

Comment:
What I have done is inverted the logic in the function so that it is consistent with it's name. I have fixed all references to the function to now used the opposite logic. Function now returns true on success.